schema/game.json

```diff
@@ -0,0 +1,238 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "definitions": {
+    "url": {
+      "type": "string",
+      "pattern": "^(http|https|ftp)://([a-zA-Z\\.\\d/#?=%+_]+)$"
+    },
+    "type_name": {
+      "description": "CamelCase type name",
+      "type": "string",
+      "pattern": "^[A-Z][a-zA-Z0-9]+$"
+    },
+    "extension": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "pattern": "^[a-zA-Z0-9_\\-]+$"
+        },
+        "commands": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/command"
+          }
+        }
+      },
+      "required": [ "name", "commands" ],
+      "additionalProperties": false
+    },
+    "command": {
+      "description": "Command definition",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Opcode unique identifier",
+          "type": "string",
+          "pattern": "^[0-9a-fA-F]{4}$"
+        },
+        "name": {
+          "description": "Opcode unique name (Rockstar fashion)",
+          "pattern": "^([A-Z0-9_]+|{|})$"
+        },
+        "num_params": {
+          "description": "Total parameter count",
+          "type": "integer",
+          "minimum": 0,
+          "maximum": 20
```

Comment:
The maximum is unset, however normally it would be 16 params in 3/VC and 32 in San Andreas

Reply:
Oops, it's my mistake. It should be increased. I wanted to add reasonable limit to make schema stronger.